include/z64.h

```diff
@@ -756,6 +758,7 @@ typedef struct {
     /* 0x38 */ DmaRequest dmaRequest;
     /* 0x58 */ OSMesgQueue loadQueue;
     /* 0x70 */ OSMesg loadMsg;
+    /* 0x74 */ s16 unk_74[2];
 } RoomContext; // size = 0x74
```

Comment:
Are you sure about this? The size doesn't match

Reply:
oh i forgot to update it lol